docs/getting-started-guides/ubuntu/automated.md

```diff
@@ -0,0 +1,253 @@
+---
+assignees:
+- caesarxuchao
+- erictune
+
+---
+
+Ubuntu 16.04 introduced the [Canonical Distribution of Kubernetes](https://jujucharms.com/canonical-kubernetes/), a pure upstream distribution of Kubernetes designed for production usage. Out of the box it comes with the following components on 12 machines:
+
+- Kubernetes (automated deployment, operations, and scaling)
+     - Three node Kubernetes cluster with one master and two worker nodes.
+     - TLS used for communication between nodes for security.
+     - Flannel Software Defined Network (SDN) plugin
+     - A load balancer for HA kubernetes-master (Experimental)
+     - Optional Ingress Controller (on worker)
+     - Optional Dashboard addon (on master) including Heapster for cluster monitoring
+- EasyRSA
+     - Performs the role of a certificate authority serving self signed certificates
+       to the requesting units of the cluster.
+- Etcd (distributed key value store)
+     - Three node cluster for reliability.
+- Elastic stack
+     - Two nodes for ElasticSearch
+     - One node for a Kibana dashboard
+     - Beats on every Kubernetes and Etcd node:
+          - Filebeat for forwarding logs to ElasticSearch
+          - Topbeat for inserting server monitoring data to ElasticSearch
+
+
+The Juju Kubernetes work is curated by a dedicated team of community members,
+let us know how we are doing. If you find any problems please open an
+[issue on the kubernetes project](https://github.com/kubernetes/kubernetes/issues)
```

Comment:
We have been funneling users to the bundle issue tracker, we might want to keep that notion here so we don't wind up managing issues in 2 places. We're likely to be missing bugs if we don't normalize that location.